openpype/lib/transcoding.py

```diff
@@ -1037,3 +1037,56 @@ def convert_ffprobe_fps_to_float(value):
     if divisor == 0.0:
         return 0.0
     return dividend / divisor
+
+
+def convert_colorspace(
+    input_path,
+    out_filepath,
+    config_path,
+    source_colorspace,
+    target_colorspace,
+    logger=None
+):
+    """Convert source files from one color space to another.
+
+    Filenames of input files are kept so make sure that output directory
+    is not the same directory as input files have.
+    - This way it can handle gaps and can keep input filenames without handling
+        frame template
+
+    Args:
+        input_path (str): Paths that should be converted. It is expected that
+            contains single file or image sequence of samy type.
+        out_filepath (str): Path to directory where output will be rendered.
+            Must not be same as input's directory.
+        config_path (str): path to OCIO config file
+        source_colorspace (str): ocio valid color space of source files
+        target_colorspace (str): ocio valid target color space
+        logger (logging.Logger): Logger used for logging.
+
+    """
+    if logger is None:
+        logger = logging.getLogger(__name__)
+
+    input_arg = "-i"
+    oiio_cmd = [
+        get_oiio_tools_path(),
+
+        # Don't add any additional attributes
+        "--nosoftwareattrib",
+        "--colorconfig", config_path,
+        "--colorconvert", source_colorspace, target_colorspace
+    ]
+    # Prepare subprocess arguments
+
+    oiio_cmd.extend([
+        input_arg, input_path,
+    ])
+
+    # Add last argument - path to output
+    oiio_cmd.extend([
+        "-o", out_filepath
+    ])
```

Comment:
```suggestion
        # Input arguments
        "-i", input_path,

        # Output arguments
        "-o", out_filepath
    ]
```

Wouldn't this be just as readable if not more readable?

Reply:
Not part of this PR. Comes from; https://github.com/ynput/OpenPype/pull/4291